cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ActionExecutionData.java

```diff
@@ -136,6 +148,30 @@ public void doAction(@Param(EXECUTION_RUNTIME_SERVICES) ExecutionRuntimeServices
         runEnv.putNextStepPosition(nextStepId);
     }
 
+    private Map<String, Value> runSequentialAction(
+            Map<String, SerializableSessionObject> serializableSessionData,
+            Map<String, Value> currentContext,
+            Map<String, Map<String, Object>> nonSerializableExecutionData,
+            String gav,
+           List<SeqStep> seqSteps, String nodeNameWithDepth,
+            int depth) {
+        @SuppressWarnings("unchecked")
+        Map<String, Serializable> returnMap =
+                (Map<String, Serializable>)
+                        seqExecutionService.execute(
+                                gav,
+                                new CloudSlangSequentialExecutionParametersProviderImpl(serializableSessionData,
+                                        currentContext,
+                                        nonSerializableExecutionData,
+                                        seqSteps,
+                                        nodeNameWithDepth,
+                                        depth));
+        if (returnMap != null) {
```

Comment:
Use ternary operator. 
Makes sense to throw an exception to be consistent with previous implementation in case the map, as the return map should not be empty.